OloEngine/src/OloEngine/Renderer/Commands/CommandBucket.cpp

```
@@ -736,43 +737,16 @@ namespace OloEngine
         m_IsBatched = false;
     }
 
-    u32 CommandBucket::RegisterWorkerThread()
+    void CommandBucket::UseWorkerIndex(u32 workerIndex)
```

Cppcheck (2.19.0)

[style] The function 'UseWorkerIndex' is never used.

(unusedFunction)